src/screens/FormScreen.js

```diff
@@ -85,8 +85,10 @@ export const FormScreen = () => {
             <Mutation mutation={createQuery(QUERY_TYPES.APP_USER_CONTENT)}>
               {(createAppUserContent) => (
                 <View style={{ padding: normalize(14) }}>
+                  <BoldText>Name</BoldText>
                   <BoldText>Name</BoldText>
                   <TextInput
+                    accessibilityLabel={`(Geben Sie Ihren Namen ein) ${name}`}
```

Comment:
Is a sentence like that the way to go with accessibility? (i don't know how other apps solve this)

i would have expected something along the line of `Eingabefeld (Name): ${name}`

Reply:
I've studied about accessibility labels ( unfortunately I lost the article ) and I've learned that for the most frequent commands is right to give a short description, in order to make the user experience more fluent. But in the case of a "rare" command where the user is required to perform a very precise action, you can choose a "friendly" label. 
What do you guys think about it ?